internal/main_vm.go

```diff
@@ -31,6 +34,27 @@ func Main() {
 	}
 }
 
+// Find the path to package main by looking at the root Caller.
+func findMainPath() string {
+	pc := make([]uintptr, 20)
+	_ = runtime.Callers(0, pc)
+	frames := runtime.CallersFrames(pc)
+	for {
+		frame, more := frames.Next()
+		if frame.File == "" {
+			break
+		}
+		// Tests won't have package main, instead they have testing.tRunner
+		if frame.Function == "main.main" || frame.Function == "testing.tRunner" {
```

Comment:
There isn't any guarantee about the format of this string AFAICT, so I think a more reliable signal is to use the last frame.